chaincode/compute_plan.go

```diff
@@ -392,6 +392,7 @@ func UpdateComputePlan(db *LedgerDB, ComputePlanID, tupleStatus, tupleKey string
 		return err
 	}
 	if cp.CheckNewTupleStatus(tupleStatus) {
+		db.AddComputePlanEvent(ComputePlanID, cp.Status)
```

Comment:
Have you considered renaming this function? `UpdateComputePlanStatus` seems more accurate IMO.

Reply:
**Edit**: this comment is about the `CheckNewTupleStatus` function!